src/components/layout/footer.js

```diff
@@ -1,9 +1,233 @@
 import React from 'react'
+import { Link } from 'gatsby'
 import { localize } from '../localization'
+import styled from 'styled-components'
+import device from 'themes/device'
+import Container from '../containers/container'
+import Logo from '../../images/svg/logo.svg'
+import YouTube from "../../images/svg/youtube.svg";
+import Twitter from "../../images/svg/twitter.svg";
+import Telegram from "../../images/svg/telegram.svg";
+import Reddit from "../../images/svg/reddit.svg";
+import Facebook from "../../images/svg/social-facebook.svg";
+import Labuan from "../../images/svg/footer-labuan.svg";
+import FSC from "../../images/svg/fsc.svg";
+import Vanuatu from "../../images/svg/footer-vanuatu.svg";
+import Warning from "../../images/svg/warning.svg";
+
+const FooterNavGrid = styled.div`
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    width: 100%;
+
+    @media ${device.laptop} {
+        grid-template-columns: 1fr;
+        grid-row-gap: 2rem;
+    }
+`
+const FooterNav = styled.div`
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 1rem;
+    width: 100%;
+`
+const FooterSocket = styled.section`
+    background-color: var(--color-white);
+    padding: 2rem 0;
+
+    h4 {
+        text-transform: uppercase;
+        font-size: 1.4rem;
+        font-weight: 500;
+        margin-bottom: 2rem;
+    }
+`
+const Legal = styled.section`
+    background-color: var(--color-grey-1);
+    color: var(--color-grey-3);
+    font-size: 1.2rem;
+    width: 100%;
+    padding: 2rem 0;
+
+    p {
+        margin-bottom: 1.8rem;
+        line-height: 1.8rem;
+    }
+`
+const LegalRow = styled.div`
+    display: grid;
+    grid-template-columns: 3fr 2fr;
+    grid-column-gap: 2rem;
+    max-width: 100%;
+
+    @media ${device.laptop} {
+        grid-template-columns: 1fr;
+        grid-column-gap: 0;
+    }
+
+    div * {
+        align-self: center;
+        justify-self: center;
+    }
+    span {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+    }
+`
+const RiskNote = styled.section`
+    background-color: var(--color-black);
+    padding: 1.4rem 0; 
+    color: var(--color-white);
+    line-height: 1.2rem;
+
+    p {
+        font-size: 1.2rem;
+        line-height: 1.8rem;
+        text-align: justify;
+    }
+    h4 {
+        text-transform: uppercase;
+        font-size: 1.4rem;
+        font-weight: bold;
+        letter-spacing: 1px;
+        margin-bottom: 1.1rem;
+    }
+`
+const StyledLink = styled(props => <Link {...props} />)`
```

Comment:
Please use LocalizedLink instead (will be necessary once we enable routing for languages)
```suggestion
const StyledLink = styled(props => <LocalizedLink {...props} />)`
```